Packages/com.chisel.editor/Chisel/Editor/Editor/Scene/Handles/PointDrawing.cs

```diff
@@ -203,6 +235,20 @@ public static void Do(int id, Rect dragArea, ref List<Vector3> points, out Matri
                         using (new UnityEditor.Handles.DrawingScope(selectedColor))
                         {
                             HandleRendering.RenderSnapping3D(s_Snapping2D.WorldSlideGrid, s_Snapping2D.WorldSnappedExtents, s_Snapping2D.GridSnappedPosition, s_Snapping2D.SnapResult, true);
+
+                            using (new UnityEditor.Handles.DrawingScope(s_Transform))
+                            {
+                                var count = points.Count - 1;
+                                for (int i = 0; i < count - 1; i++)
+                                {
+                                    if ((points[count] - points[i]).sqrMagnitude < kDistanceEpsilon)
+                                    {
+                                        if (i > 0)
+                                            UnityEditor.Handles.color = Color.red;
+                                        capFunction(-1, points[count], orientation, UnityEditor.HandleUtility.GetHandleSize(points[count]) * (kPointScale * 2.0f), type);
```

Comment:
Render a 'highlight' of sorts when the current point is hovering another point, make it red (invalid) when it's not the first point (which will close the loop)